Formula/civl.rb

```diff
@@ -40,7 +46,7 @@ def install
       }
     EOS
     # Test with example suggested in manual.
-    example = pkgshare/"examples/concurrency/locksBad.cvl"
+    example = testpath/"examples/concurrency/locksBad.cvl"
```

Comment:
Can we just install this fixture into `pkgshare` so we don't need a resource that's identical to the stable tarball?

Reply:
In the install block? Won't the audit complain about mismatched binaries being shipped with the bottle?

Reply:
Probably not.
```
❯ file locksBad.cvl
locksBad.cvl: C source, ASCII text
```

Reply:
Done. I misread your comment. Should be good now.